src/OSPSuite.Core/Domain/EntitySource.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using OSPSuite.Utility.Collections;
+using OSPSuite.Utility.Visitor;
+
+namespace OSPSuite.Core.Domain
+{
+   /// <summary>
+   ///    Tracks the origin of specific entities in the simulation (such as parameters, initial conditions, parameter values
+   ///    and formulas).
+   ///    The origin module will be the one that last modified the entity (e.g a parameter might be created by multiple
+   ///    module. This will be the one that created the entity)
+   ///    Each source is unique by path. It will store the id of the module, potentially the id of the building block, the
```

Comment:
Module Id is not included anymore. I think it was at one time. Is there a source where the building block is not stored since it is stored 'potentially'?

Reply:
yeah this is oudated. I'll update. the documentation